src/batchedlogger.cpp

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2018-2019 The Dash Core developers
+// Copyright (c) 2018-2020 The Dash Core developers
```

Comment:
Why do some files (e.g. this one) use 2020 while others use 2021? Is it based on when the file was last modified?

Reply:
https://github.com/dashpay/dash/blob/master/contrib/devtools/copyright_header.py#L415